pkgs/development/libraries/gourou/base64.patch

```diff
@@ -0,0 +1,143 @@
+diff --git a/include/base64.h b/include/base64.h
+new file mode 100644
+index 000000000000..cdfdc04d0bb1
+--- /dev/null
++++ b/include/base64.h
```

Comment:
This is not a patch, it is a file inclusion disguised as patch.

Why?

Reply:
Because it's a [gist](https://gist.github.com/f0fd86b6c73063283afe550bc5d77594) [fetched at build time](https://indefero.soutade.fr/p/libgourou/source/tree/master/scripts/setup.sh#L13) otherwise.  I also sent the same patch upstream.

Reply:
Since it is a gist, you can just fetch it using `fetchgit` or even with `fetchzip` (by using `https://gist.github.com/<username>/<gist>/archive/<commit>.zip`) instead of doing this. Them just copy it after the `unpackPhase` inside the build directory.

This will avoid committing a big "patch" inside nixpkgs repository, and also will allow tracking of its origins better.

Reply:
Thanks, done!